source/win-stats/GraphView.vue

```diff
@@ -185,7 +197,12 @@ onMounted(() => {
   }
 
   controlsObserver.observe(controlsElement.value, { box: 'border-box' })
-
+  availableFolders.value = {
+  'All folders': '',
+  'Zettlr Tutorial': '/zettlr-tutorial',
+  'Tutorial Notes': '/zettlr-tutorial/notes',
+  'Tutorial Research': '/zettlr-tutorial/research'
+}
```

Comment:
Do I see it correctly that you literally just hardcoded these options and there is no functionality in your PR to retrieve the actual folders that a user has?